### PR TITLE
Add support for redis 5.x

### PR DESCRIPTION
**Pull request by tamirOK**
`HiredisParser` changed to `_AsyncHiresParser`
`PythonParser` changed to `_AsyncRESP2Parser`

Implementations are exactly the same

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def _get_long_description():
     install_requires=[
         'click>=6.7,<9',
         'pydantic>=1,<2',
-        'redis>=4.2,<5',
+        'redis>=4.2,<6',
```
Comment (martyanov) [inline setup.py:58]: Suppor for verstion < 5 is broken now, you can use aliases.

Comment (tamirOK) [reply setup.py:58]: Made changes backwards compatible

File: arku/parser.py [hunk 2]
```diff
@@ -5,9 +5,15 @@
 from contextvars import ContextVar
 from typing import Any, Dict, Generator, Optional, Type, Union
 
-from redis.asyncio.connection import SERVER_CLOSED_CONNECTION_ERROR, Connection, Encoder, HiredisParser, PythonParser
+from redis.asyncio.connection import Connection, Encoder
 from redis.typing import EncodableT, EncodedT
 
+try:
```
Comment (martyanov) [inline arku/parser.py:11]: It is better to separate the imports, one import - one fallback.

Comment (tamirOK) [reply arku/parser.py:11]: Fixed

**Review by martyanov — APPROVED** [review]

**Review by martyanov — APPROVED** [review]

**Pull request merged**